doc/man3/ASN1_INTEGER_new.pod

```diff
@@ -0,0 +1,44 @@
+=pod
+
+=head1 NAME
+
+ASN1_INTEGER_new, ASN1_INTEGER_free, ASN1_INTEGER_it -
```

Comment:
In spite of listing `ASN1_INTEGER_it` here it  doesn't say anything about this type on this page.

Reply:
fixed

Reply:
I don't see a fix for this?

Reply:
fix commited